### ISSUE TITLE
Java Records not supported?

### ISSUE THREAD

**Issue by mudlee** [Cosium/git-code-format-maven-plugin#68]
Hi.

I'm trying to validate my code with this great plugin. However, for my `record` files I get this:
```bash
[ERROR] Failed to execute goal com.cosium.code:git-code-format-maven-plugin:2.7:on-pre-commit (default-cli) on project api: com.google.googlejavaformat.java.FormatterException: 7:9: error: class, interface, or enum expected -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException
```

Where can I turn on the `--enable-preview` flag?

**Comment by reda-alaoui** [comment]
Looks like Record formatting is available since gjf 1.8. 
Therefore, please try with version 3.0 of this plugin.

**Comment by mudlee** [comment]
Ah, it was just release 2 days ago :D Ok, with 3.0 it works! Thanks :)
